ProductTree/Views/ProductTree.xaml

```diff
@@ -184,7 +134,8 @@
                                   NavigationStyle="Cell"
                                   ShowHorizontalLines="False"
                                   ShowIndicator="False"
-                                  ShowNodeImages="False"
+                                  ShowNodeImages="True"
```

Comment:
Didnt have to add here `NodeImageSize="20,16"` ?

Reply:
Not needed here